tasks/setup/default.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Set variable with services List for RedHat 7 and above systems 
```

Comment:
yamllint is complaining about trailing spaces - I think this line has trailing spaces

Reply:
Fixed